packages/react-components/src/components/AttachmentDownloadCards.tsx

```diff
@@ -39,19 +50,17 @@ export interface _AttachmentDownloadCardsStrings {
  */
 export interface _AttachmentDownloadCardsProps {
   /**
-   * User id of the local participant
+   * A chat message metadata that includes file metadata
    */
-  userId: string;
+  attachments?: AttachmentMetadata[];
   /**
    * A chat message metadata that includes file metadata
    */
-  fileMetadata?: AttachmentMetadata[];
+  message?: ChatMessage;
   /**
-   * A function of type {@link FileDownloadHandler} for handling attachment downloads.
-   * If the function is not specified, the attachment's `url` will be opened in a new tab to
-   * initiate the download.
+   * Optional callback to handle file download
    */
-  downloadHandler?: FileDownloadHandler;
+  actionForAttachment?: (attachment: AttachmentMetadata, message?: ChatMessage) => AttachmentMenuAction[];
```

Comment:
```suggestion
  actionsForAttachment?: (attachment: AttachmentMetadata, message?: ChatMessage) => AttachmentMenuAction[];
```
Multiple actions on the array.